niceman/support/external_versions.py

```diff
@@ -69,7 +72,8 @@ class ExternalVersions(object):
 
     CUSTOM = {
         'cmd:annex': _get_annex_version,
-        'cmd:git': _get_git_version
+        'cmd:git': _get_git_version,
+        'cmd:apt-cache': _get_apt_cache_version
```

Comment:
insignificant "styling" comment:  python allow trailing `,` between items of the list, dict, or tuple.  So I usually add it to minimize the diff next time (since I wouldn't need to add it to preceding line)